src/components/CommonComponents/SearchBar/index.tsx

```diff
@@ -91,15 +91,23 @@ const SearchBar = (): JSX.Element => {
     }
   };
 
+  const toggleContainer = () => {
```

Comment:
I know these functions return `void` but wdyt:

```js
const toggleContainer = () =>
  isExpanded ? collapseContainer : expandContainer
```

Reply:
Yea we can change